lumibot/strategies/strategy_executor.py

```diff
@@ -632,8 +631,10 @@ def _strategy_sleep(self):
                     # Remove the time to close from the strategy sleep time.
                     strategy_sleeptime -= time_to_close
 
-                    # Process expired option contracts.
-                    self.broker.process_expired_option_contracts(self.strategy)
+                    # Check if the broker has a function to process expired option contracts.
+                    if hasattr(self.broker, "process_expired_option_contracts"):
+                        # Process expired option contracts.
+                        self.broker.process_expired_option_contracts(self.strategy)
```

Comment:
Should we add this as a required @abstactmethod?  Brokers that don't care, can just define a "pass" version of their own, but I feel like if this is just buried so deep in the code, it will be forgotten about.

Reply:
process_expired_option_contracts is only for the backtesting broker, no other broker will ever need this method